src/libraries/Common/src/Interop/Windows/BCrypt/Interop.BCryptGenerateKeyPair.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class BCrypt
+    {
+        [LibraryImport(Libraries.BCrypt)]
+        private static unsafe partial NTSTATUS BCryptGenerateKeyPair(
+            SafeBCryptAlgorithmHandle hAlgorithm,
+            out SafeBCryptKeyHandle phKey,
+            int dwLength,
```

Comment:
We don't do it consistently, but should this be a uint to match the DWORD?

Reply:
It marshals out the same, and requires adding a cast; so doesn't really seem worth it?

Reply:
That's fine but there seems to be difference of opinion on it even in terms of style. e.g. https://github.com/dotnet/runtime/pull/33820#issuecomment-601423697.

I've been under the impression "we don't like it for net-new code" so I'm really just asking to clarify my own understanding.